kbi/0014/index.rst

```diff
@@ -0,0 +1,49 @@
+.. index::
+   single: datalad push; tags
+
+KBI0014: Pushing tags with ``datalad push``
+===========================================
+
+:authors: Michael Hanke <michael.hanke@gmail.com>, Stephan Heunis <jsheunis@gmail.com>
+:discussion: https://github.com/psychoinformatics-de/knowledge-base/pull/52
+:keywords: datalad push, tags, git config
+:software-versions: datalad_0.18.3, git_2.39.1
+
+The `datalad push`_ command makes a saved state of a dataset available to a sibling, both
+in terms of the dataset's git repository and its annex. Internally, the command uses `git push`_
+and ``git annex copy`` to push a dataset state to a sibling. Importantly, it honors the configuration
+of ``git push`` by allowing users to specify e.g. *what* to push, which could include the repository tags.
+This implies that if the datalad dataset (i.e. git repository) is configured via `git config`_
+to include tags in the push operation, the ``datalad push`` command will result in the tags being pushed
+to the sibling.
+
+The example code below demonstrates how to configure a dataset's so-called ``<refspec>``
```

Comment:
```suggestion
The example code below demonstrates how to configure a dataset's so-called `refspec <https://git-scm.com/book/en/v2/Git-Internals-The-Refspec>`_
```

Maybe make this a link so that readers can go find out more?